### ISSUE TITLE
[automation] Rules are added twice after modifying JSR223 (Python) files

### ISSUE THREAD

**Issue by cweitkamp** [openhab/openhab-core#2246]
After modification of textual Rule files (Python) the Rules are created twice:

```
openhab> automation listRules | grep Energy
20     08db0ffa-a6f7-44fe-8404-42a0226f8124 Energy - Trigger for sensorTerraceEnergy changed IDLE           
182    6d24f9ae-c787-4452-b623-b82727d6063d Energy - Trigger for sensorTerracePower changed IDLE           
258    97d39ee5-b517-4377-bb37-928efb1a193c Energy - Trigger for sensorTerracePower changed IDLE           
278    abfa6e83-cf1b-4ec3-8242-7d5aefc86b13 Energy - Trigger for sensorTerraceEnergy changed IDLE 
```

![grafik](https://user-images.githubusercontent.com/5131747/111864466-5244b680-8961-11eb-8630-4ccbe1a7f75c.png)

// CC @jpg0 Do you think this could be related to #2222? I observed this issue since a few days. I am not aware ti was happening before.

**Comment by jpg0** [comment]
hi @cweitkamp, I just updated my system to the latest snapshot (including #2222) to take a look at this. I am seeing the same behaviour; I'll try to figure out what is causing it...

**Comment by jpg0** [comment]
Found it, you're right, it was #2222. Sorry about that! PR to fix is #2254.
